cachi2/core/package_managers/general.py

```diff
@@ -44,6 +56,94 @@ def download_binary_file(
             f.write(chunk)
 
 
+async def _async_download_binary_file(
+    session: aiohttp_retry.RetryClient,
+    url: str,
+    download_path: Union[str, PathLike[str]],
+    auth: Optional[aiohttp.BasicAuth] = None,
+    chunk_size: int = 8192,
+) -> None:
+    """
+    Download a binary file (such as a TAR archive) from a URL using asyncio.
+
+    :param aiohttp_retry.RetryClient session: Aiohttp interface for making HTTP requests.
+    :param str url: URL for file download
+    :param str download_path: File path location
+    :param aiohttp.BasicAuth auth: Authentication for the URL
+    :param int chunk_size: Chunk size param for Response.content.read()
+    :raise FetchError: If download failed
+    """
+    try:
+        log.debug(f"Download started - {url}")
```

Comment:
What do you think about removing this log here and keeping the completed/unsuccessful ones? Even though they're DEBUG level, these logs can get pretty verbose.

This would make this work similarly to Pip: 
- 1 log for each downloaded dep
- 1 log for each validated checksum

Reply:
I don't mind either way

Reply:
Sure I can remove this log so we have similar logs to pip.

Reply:
removed